envpool/atari/atari_env.h

```diff
@@ -124,40 +131,67 @@ class AtariEnv : public Env<AtariEnvSpec> {
     }
     // init buf
     for (int i = 0; i < 2; ++i) {
-      maxpool_buf_.push_back(std::move(Array(raw_spec_)));
+      maxpool_buf_[i].reset(new uint8_t[kRawSize]);
     }
     for (int i = 0; i < stack_num_; ++i) {
       stack_buf_.push_back(Array(resize_spec_));
     }
     gray_obs_ = Array(FrameSpec({kRawHeight, kRawWidth, 1}));
-    ResetObsBuffer();
```

Comment:
Why we don't need to reset zero out the buffer anymore?

Reply:
because I use push_all=True flag in PushStack